src/plugins/dev_tools/public/dev_tool.ts

```diff
@@ -29,9 +28,14 @@ export class DevToolApp {
    * This will be used as a label in the tab above the actual tool.
    * May also be a ReactNode.
    */
-  public readonly title: ReactNode;
+  public readonly title: string;
   public readonly mount: AppMount;
 
+  /**
+   * Mark the navigation tab as beta.
+   */
+  public readonly isBeta?: boolean;
```

Comment:
We already had app titles that we could use for setting the breadcrumbs, tabs and doc title, but the problem was that `painlessLab` was returning a ReactNode since it was also adding the `<EuiBeta>` component to it. I ended up refactoring it so that the title is always a string that we can use in all 3 places, but there's now an extra prop called `isBeta` that allows us to add the `<EuiBeta>` component to the tabs